build.gradle

```diff
@@ -1,6 +1,28 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.connexta'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
```

Comment:
❓ Why is this `configurations` section needed?

Reply:
It was added when I included the Lombok dependency. The `configurations` section is used to scope dependencies, similar to how Maven does it.